### PR TITLE
Add SIG Instrumentation annual report, 2021

### LINKED ISSUE #5504: 2021 Annual Report: SIG Instrumentation
Chairs: @ehashman @logicalhan 
Liasion: @cblecker

- [ ] Consult SIG to complete draft of report
- [ ] If needed, consult with Steering Committee liasion on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your SC liasion for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, SC will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

Key dates:
- Initial PR to communtiy repo should be opened by March 1st, 2021
- PR should be reviewed and merged by April 1st, 2021

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liasion directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig instrumentation
/committee steering

### PR DESCRIPTION
**Pull request by ehashman**
**Which issue(s) this PR fixes**:

Fixes #5504.

/hold
for discussion

/sig instrumentation
/cc @kubernetes/sig-instrumentation-members

### CODE REVIEW

File: sig-instrumentation/annual-report-2021.md
Comment (ehashman) [inline sig-instrumentation/annual-report-2021.md:70]: @logicalhan can we add a link for promtool? not sure what it is, structured logging is linked above

**Comment by ehashman** [comment]
/cc @cblecker 
instrumentation steering liaison

File: sig-instrumentation/annual-report-2021.md
Comment (kakkoyun) [reply sig-instrumentation/annual-report-2021.md:70]: This should help https://github.com/prometheus/prometheus/tree/main/cmd/promtool

Comment (logicalhan) [reply sig-instrumentation/annual-report-2021.md:70]: I think she was referring (?) to promq, which is something we wrote. 

https://github.com/kubernetes-sigs/instrumentation-tools

(Sorry have been OOO)

Comment (logicalhan) [reply sig-instrumentation/annual-report-2021.md:70]: https://github.com/kubernetes-sigs/instrumentation-tools/tree/master/promq

**Review by logicalhan — APPROVED** [review]
/lgtm
/approve

**Review by lilic — APPROVED** [review]
/lgtm
/approve

🎉 Thanks for taking care of this!

**Comment by ehashman** [comment]
I will remove the hold on Friday, Mar. 19 after a week of lazy consensus if no further comments.

File: sig-instrumentation/annual-report-2021.md
Comment (lilic) [inline sig-instrumentation/annual-report-2021.md:63]: Not sure if kube-state-metrics 2.0 efforts should be mentioned here? @brancz ?

Comment (brancz) [reply sig-instrumentation/annual-report-2021.md:63]: yes!

**Comment by serathius** [comment]
@dgrisonnet @s-urbaniak Should we also mention work on [Metrics Server](https://github.com/kubernetes-sigs/metrics-server/issues/627) and [prometheus adapter](https://github.com/kubernetes-sigs/prometheus-adapter) ?

File: sig-instrumentation/annual-report-2021.md
Comment (lilic) [reply sig-instrumentation/annual-report-2021.md:63]: Maybe:
```
* cut the release of major breaking v2.0.0 of kube-state-metrics 
```

**Comment by dgrisonnet** [comment]
:+1: sounds like a good idea, I can prepare something for prometheus-adapter.

File: sig-instrumentation/annual-report-2021.md
Comment (dgrisonnet) [reply sig-instrumentation/annual-report-2021.md:63]: Should this really be mentioned under the `Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?` part?

Maybe we could add a whole new bullet point for the SIG sub projects and add a few words about kube-state-metrics there?

Comment (lilic) [reply sig-instrumentation/annual-report-2021.md:63]: Yes this was just a random point I added the comment, agreed its up to @ehashman where to place it.

**Comment by ehashman** [comment]
Please each give me a one line blurb + link :)

**Comment by lilic** [comment]
> * kube-state-metrics is on its way to a new major breaking v2.0.0 release, with the v2.0.0-rc.0 being released recently. https://github.com/kubernetes/kube-state-metrics/releases/tag/v2.0.0-rc.0

@brancz can suggest otherwise.

**Comment by s-urbaniak** [comment]
agreed on adding a line about prometheus-adapter, metrics-server, maybe even custom-metrics-apiserver :+1:

**Comment by dgrisonnet** [comment]
> - custom-metrics-apiserver is shepherding a new initiative to support multiple custom and external metrics servers. https://github.com/kubernetes-sigs/custom-metrics-apiserver/issues/70
> - prometheus-adapter recently joined the kubernetes-sigs organization and is still being integrated. The next important step for the project is its complete overhaul to improve its scability and make its configuration more user friendly. https://github.com/s-urbaniak/prometheus-adapter/blob/master/design.md

Here are some suggestions for custom-metrics-apiserver and prometheus-adapter, let me know if it's fine with you.

**Comment by ehashman** [comment]
Added subproject updates.

/hold cancel

LGTM at will :)

**Comment by cblecker** [comment]
retroactive LGTM

File: sig-instrumentation/annual-report-2021.md
Comment (nikhita) [inline sig-instrumentation/annual-report-2021.md:16]: I'm probably missing something here...can you point me to the process for keeping OWNERS up to date?

File: sig-instrumentation/annual-report-2021.md
Comment (nikhita) [inline sig-instrumentation/annual-report-2021.md:36]: :heart: this

**Review by nikhita — COMMENTED** [review]
Retroactive lgtm. Just left one question, but it's probably me missing something here :)

File: sig-instrumentation/annual-report-2021.md
Comment (ehashman) [reply sig-instrumentation/annual-report-2021.md:16]: Ah, there were a bunch of scratch notes that didn't make it in here. This was referring to https://github.com/kubernetes/community/pull/5603/files#diff-c7b01de0190da099e3c4819f213c78276bb6514a43f9f51339277f65c7ee7b29R10-R11

When we went through reviewing our charter and subprojects, we also reviewed all our OWNERS files, which are mainly in subprojects as we only own a small portion of k/k.

Comment (ehashman) [reply sig-instrumentation/annual-report-2021.md:16]: The updates were https://github.com/kubernetes/kubernetes/pull/99013 and https://github.com/kubernetes/org/pull/2497 https://github.com/kubernetes/org/pull/2541

**Pull request merged**
